deploy/auth-user.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: metrics-server:system:anonymous
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: metrics-server
+subjects:
+- kind: User
+  name: system:anonymous
```

Comment:
I'd like to have access to metrics through apiserver. Without this I wasn't able to do it.
@deads2k @cheftako what should I do instead here?